core-bundle/src/Command/InstallCommand.php

```diff
@@ -239,8 +246,10 @@ private function addInitializePhp()
      */
     private function symlinkTcpdfConfig()
     {
+        $relPath = $this->fs->makePathRelative($this->bundlesMeta['ContaoCoreBundle']['path'], $this->rootDir);
+
         SymlinkUtil::symlink(
-            'vendor/contao/core-bundle/src/Resources/contao/config/tcpdf.php',
+            $relPath.'Resources/contao/config/tcpdf.php',
```

Comment:
Shouldn‘t there be a slash here?

Reply:
`makePathRelative` always returns a trailing slash I think. But may be we should not rely on that and use `rtrim($relPath, '/').'/…'`?

Reply:
Why not use `Webmozart\PathUtil\Path` for all of these operations (making things relative, joing paths). It's already a dependency anyways. Doing it by hand is so error prone...

Reply:
Using `Path::join` now ;). Whether we use `Filesystem::makePathRelative` or `Path::makeRelative` … 🤷‍♂️

Reply:
If you're using the filesystem anyways that doesn't really matter I guess. Otherwise I prefer the `Path::*` methods because of their static nature - this never has to be a dependency (it's fine to test in place with real paths).